web/web-itests/pom.xml

```diff
@@ -47,13 +47,12 @@
         <logback.version>1.5.16</logback.version>
         <org.apache.aries.blueprint.sample.version>1.0.0</org.apache.aries.blueprint.sample.version>
         <org.apache.aries.blueprint.version>1.0.0</org.apache.aries.blueprint.version>
-        <org.apache.aries.proxy.version>1.0.1</org.apache.aries.proxy.version>
+        <org.apache.aries.proxy.version>1.1.14</org.apache.aries.proxy.version>
         <org.apache.aries.testsupport.unit.version>2.0.0-SNAPSHOT</org.apache.aries.testsupport.unit.version>
-        <org.apache.aries.util.version>1.0.0</org.apache.aries.util.version>
+        <org.apache.aries.util.version>1.2.0-SNAPSHOT</org.apache.aries.util.version>
         <org.apache.aries.web.urlhandler.version>1.0.1-SNAPSHOT</org.apache.aries.web.urlhandler.version>
-        <org.apache.felix.configadmin.version>1.2.4</org.apache.felix.configadmin.version>
-        <org.eclipse.osgi.version>3.11.3</org.eclipse.osgi.version>
-        <slf4j-api.version>1.7.7</slf4j-api.version>
+        <org.apache.felix.configadmin.version>1.9.26</org.apache.felix.configadmin.version>
+        <org.eclipse.osgi.version>3.16.0</org.eclipse.osgi.version>
```

Comment:
I tried to set version the newest version 3.22.0 as it is in `quiesce-manager-itest`, but the container is not starting with error:
```
Security class "org.osgi.service.log.LogLevel"'s signer information does not match signer information of other classes in the same package
```

Reply:
When order of dependencies is changed then this error dissappears but the highest eclipse version to use now is 3.17.0
Higher versions has WABs generated with missing headers

Reply:
The cause in the new versions of eclipse seems to be that url is:
```
webbundle:file:/tmp/test7207230737428988734.war?Bundle-SymbolicName=test.war.bundle&Web-ContextPath=foo
```
and new eclipse ignores generated input stream because `sourceUrl` has protocol `file` in https://github.com/eclipse-equinox/equinox/blob/e35221a86afd24ee21b7b9d02db298a90ace1bc0/bundles/org.eclipse.osgi/container/src/org/eclipse/osgi/storage/Storage.java#L1158-L1181

Reply:
The fix is in a separate PR https://github.com/apache/aries/pull/415
@jbonofre could you take a look?